### ISSUE TITLE
Test Processor and form functionality should work with 1stPay

### ISSUE THREAD

**Issue by twjordan** [iATSPayments/com.iatspayments.civicrm#315]
I had a real tough time finding documentation on how to configure and submit test transactions with iATS and 1stPay.

After speaking on the phone w/ iATS and goemerchant support and with the developers of the plugin in issue #312 I was able to get it working and I have documented the process here: https://gist.github.com/twjordan/68451998a68f072e079536f205486007

Still, this is not ideal or even acceptable to have to use a LIVE form to do a test transaction. The process is fragile and likely to result in incorrect information in user systems and/or potentially refused payments if the payment processor is left in the incorrect configuration.

iATS customers using 1stPay should have access to the same functionality with test processors and forms as those on legacy. iATS is featured as a core processor and advertises tight integration with civiCRM as a selling point. It is somewhat misleading to discover that testing workflows is difficult and dangerous. 

Thank you!

**Comment by Edzelopez** [comment]
Hi @twjordan 

I'm having a similar issue with testing 1stPay credit cards and hoped to see your documentation. It seems however that your gist link is private, could you please grant public access to it?

**Comment by KarinG** [comment]
@Edzelopez -> the gist is that you create a New payment processor in CiviCRM and put the Sandbox credentials in the LIVE section.

One of the options we’re talking about with iATS is to have a designated CiviCRM/1stPay test account much like TEST88 on legacy.

**Comment by KarinG** [comment]
Sorry - tabbed the wrong button on my phone!

**Comment by Edzelopez** [comment]
Thanks for the reply @KarinG!

**Comment by twjordan** [comment]
Edsel, odd, I can click on this
https://gist.github.com/twjordan/68451998a68f072e079536f205486007 In a
private window and it works.

**Comment by Edzelopez** [comment]
Yes, @twjordan, the link works on a private window thanks! Will try out the recipe you provided and hopefully see if it solves the issue I've been having with testing this. Thanks for the great documentation!
